### PR TITLE
Unify naming in Docs and Web-GUI

### PR DESCRIPTION
**Pull request by olofvndrhr**
#### Description

This PR renames most uses of "Overseerr" to "Jellyseerr" in non critical places.
As this is already the case in some instances (a few settings/docs), but not all. This PR is meant to make the naming in the Docs and Web-UI consistent.

I also changed the Plex header infos at [src/utils/plex.ts](src/utils/plex.ts), this should only make a difference for new sessions. But this can be removed to be sure that it doesn't cause any problems.

Skipped were:
- Paths/Directory names
- Snap (as it is not available atm.)
- Filenames
- Functions and everything that could change the behavior of Jellyseerr
- Test cases
- Funding and similar things

If you don't want to merge theses changes, as to have better upstream compatibility for a future merge with Overseerr, you can just close this PR.
If i missed anything you can let me know and I will fix it.

### CODE REVIEW

**Comment by Uruknara** [comment]
any update on this ?

**Review by Panda260 — COMMENTED** [review]
LGTM i guess

File: docs/using-overseerr/settings/README.md [hunk 1]
```diff
@@ -216,6 +216,6 @@ Please see [Notifications](../notifications/README.md) for details on configurin
 
 ## Jobs & Cache
 
-Overseerr performs certain maintenance tasks as regularly-scheduled jobs, but they can also be manually triggered on this page.
+Jellyseerr performs certain maintenance tasks as regularly-scheduled jobs, but they can also be manually triggered on this page.
 
-Overseerr also caches requests to external API endpoints to optimize performance and avoid making unnecessary API calls. If necessary, the cache for any particular endpoint can be cleared by clicking the "Flush Cache" button.
+OverseJellyseerrerr also caches requests to external API endpoints to optimize performance and avoid making unnecessary API calls. If necessary, the cache for any particular endpoint can be cleared by clicking the "Flush Cache" button.
```
Comment (TSKsmiley) [inline docs/using-overseerr/settings/README.md:221]:
```suggestion
Jellyseerr also caches requests to external API endpoints to optimize performance and avoid making unnecessary API calls. If necessary, the cache for any particular endpoint can be cleared by clicking the "Flush Cache" button.
```

**Comment by Fallenbagel** [comment]
> to have better upstream compatibility for a future merge with Overseerr
I will be closing this PR in favour of that. Whatever is visible to the end users are already renamed to avoid confusion.

**Pull request closed without merging**